ui/src/app/view/timestamp-table/timestamp-table.component.scss

```diff
@@ -73,7 +73,25 @@ tr{
     height: 25px;
     width: 25px;
   }
+}
+
+.gridButtons{
+  display:grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.dialogueBtn{
+grid-column: 3;
```

Comment:
Should this also have a `grid-row` value like `.acceptBtn` and `.declineBtn` (or should we remove `grid-row` from those to keep all three CSS classes consistent).

Reply:
Did a work around with grid-auto-flow: column;, makes them all appear in same line without specifying which row specifically.